pcdsdevices/signal.py

```diff
@@ -1,6 +1,14 @@
 """
 Module to define ophyd Signal subclass utilities.
 """
+# Catch semi-frequent issue with scripts accidentally run from inside module
+if __name__ != 'pcdsdevices.signal':
+    raise RuntimeError(('A script tried to import pcdsdevices.signal '
```

Comment:
Double parentheses are unnecessary, FYI.

Reply:
My Python-fu is degrading. I'll fix by adding triple parentheses.

Reply:
Quadruple or octuple would be preferable 🤓